framework/src/media/voice/SpeechDetector.cpp

```diff
@@ -155,5 +157,25 @@ bool SpeechDetectorImpl::startEndPointDetect(int timeout)
 	return mEndPointDetector->startEndPointDetect(timeout);
 }
 
+bool SpeechDetectorImpl::detectEndPoint(short *sample, int numSample)
+{
```

Comment:
why don't you null-check the sample pointer?

Reply:
Fixed it.